string.c

```diff
@@ -11737,17 +11742,10 @@ sym_swapcase(int argc, VALUE *argv, VALUE sym)
 
 /*
  *  call-seq:
- *     sym.start_with?([prefixes]+)   -> true or false
- *
- *  Returns true if +sym+ starts with one of the +prefixes+ given.
- *  Each of the +prefixes+ should be a String or a Regexp.
+ *    start_with?(*string_or_regexp) -> true or false
  *
- *    :hello.start_with?("hell")               #=> true
- *    :hello.start_with?(/H/i)                 #=> true
+ *  Equivalent to <tt>self.to_s.start_with?()</tt>; see String#start_with?.
```

Comment:
This seems to show that `String#start_with?` is called with no arguments. I think we should either include the arguments

```suggestion
 *  Equivalent to <tt>self.to_s.start_with?(*string_or_regexp)</tt>; see String#start_with?.
```

or don't show the arguments at all

```suggestion
 *  Equivalent to <tt>self.to_s.start_with?</tt>; see String#start_with?.
```

I prefer the latter, where it doesn't show the arguments, WDYT?